### PR TITLE
Update "The Roundtable" staff and add VOD links

### PR DESCRIPTION
**Pull request by pishifat**
last minute staff additions and vods

### CODE REVIEW

File: wiki/Tournaments/The_Roundtable/en.md [hunk 1]
```diff
@@ -28,14 +28,19 @@ tags:
...
+| Production | ::{ flag=RU }:: [DeadRote](https://osu.ppy.sh/users/7943969), ::{ flag=GB }:: [ilw8](https://osu.ppy.sh/users/14167692), ::{ flag=US }:: Albert Chang |
 | Designer | ::{ flag=US }:: [Flezlin](https://osu.ppy.sh/users/3696423), ::{ flag=US }:: [joiechii](https://osu.ppy.sh/users/9995116) |
-| Map Pooler | ::{ flag=US }:: [Tekkito](https://osu.ppy.sh/users/7075211), ::{ flag=US }:: [D I O](https://osu.ppy.sh/users/3958619), ::{ flag=FR }:: [PLAYOSUDROID](https://osu.ppy.sh/users/8642284), ::{ flag=US }:: [fum0](https://osu.ppy.sh/users/3715823) ::{ flag=FR }:: [Ekoro](https://osu.ppy.sh/users/284905) |
-| Map Pool Playtester | ::{ flag=PH }:: [BronyPH](https://osu.ppy.sh/users/1492995) |
+| Map Pooler | ::{ flag=US }:: [Tekkito](https://osu.ppy.sh/users/7075211), ::{ flag=US }:: [D I O](https://osu.ppy.sh/users/3958619), ::{ flag=FR }:: [PLAYOSUDROID](https://osu.ppy.sh/users/8642284), ::{ flag=US }:: [ChaosRaidz](https://osu.ppy.sh/users/3715823), ::{ flag=FR }:: [Ekoro](https://osu.ppy.sh/users/284905), ::{ flag=PH }:: [BronyPH](https://osu.ppy.sh/users/1492995) |
```
Comment (Blooshing) [inline wiki/Tournaments/The_Roundtable/en.md:35]:
```suggestion
| Map Pooler | ::{ flag=US }:: [Tekkito](https://osu.ppy.sh/users/7075211), ::{ flag=US }:: [D I O](https://osu.ppy.sh/users/3958619), ::{ flag=FR }:: [Nerowo](https://osu.ppy.sh/users/8642284), ::{ flag=US }:: [ChaosRaidz](https://osu.ppy.sh/users/3715823), ::{ flag=FR }:: [Ekoro](https://osu.ppy.sh/users/284905), ::{ flag=PH }:: [BronyPH](https://osu.ppy.sh/users/1492995) |
```

There is no player associated under `PLAYOSUDROID` unless they are restricted or something.

**Review by Blooshing — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/The_Roundtable/en.md (hunk 1 above)
Comment (pishifat) [reply wiki/Tournaments/The_Roundtable/en.md]: looks like they changed name oops

File: wiki/Tournaments/The_Roundtable/en.md [hunk 2]
```diff
@@ -28,14 +28,19 @@ tags:
 | :-- | :-- |
 | Host | ::{ flag=US }:: [BTMC](https://osu.ppy.sh/users/3171691) |
 | Project Manager | ::{ flag=US }:: [Kroytz](https://osu.ppy.sh/users/2339768), ::{ flag=US }:: [pishifat](https://osu.ppy.sh/users/3178418) |
-| Referee | ::{ flag=US }:: [Tekkito](https://osu.ppy.sh/users/7075211), ::{ flag=US }:: [D I O](https://osu.ppy.sh/users/3958619), ::{ flag=US }:: [freddiiieeee](https://osu.ppy.sh/users/7112839) |
-| Production | ::{ flag=RU }:: [DeadRote](https://osu.ppy.sh/users/7943969), ::{ flag=GB }:: [Jame](https://osu.ppy.sh/users/14167692), Albert Chang |
+| Commentators | ::{ flag=US }:: [BTMC](https://osu.ppy.sh/users/3171691) ::{ flag=US }:: [HappyStick](https://osu.ppy.sh/users/256802) |
```
Comment (TPGPL) [inline wiki/Tournaments/The_Roundtable/en.md:31]:
```suggestion
| Commentator | ::{ flag=US }:: [BTMC](https://osu.ppy.sh/users/3171691) ::{ flag=US }:: [HappyStick](https://osu.ppy.sh/users/256802) |
```

File: wiki/Tournaments/The_Roundtable/en.md [hunk 3]
```diff
@@ -28,14 +28,19 @@ tags:
...
+| Production | ::{ flag=RU }:: [DeadRote](https://osu.ppy.sh/users/7943969), ::{ flag=GB }:: [ilw8](https://osu.ppy.sh/users/14167692), ::{ flag=US }:: Albert Chang |
 | Designer | ::{ flag=US }:: [Flezlin](https://osu.ppy.sh/users/3696423), ::{ flag=US }:: [joiechii](https://osu.ppy.sh/users/9995116) |
-| Map Pooler | ::{ flag=US }:: [Tekkito](https://osu.ppy.sh/users/7075211), ::{ flag=US }:: [D I O](https://osu.ppy.sh/users/3958619), ::{ flag=FR }:: [PLAYOSUDROID](https://osu.ppy.sh/users/8642284), ::{ flag=US }:: [fum0](https://osu.ppy.sh/users/3715823) ::{ flag=FR }:: [Ekoro](https://osu.ppy.sh/users/284905) |
-| Map Pool Playtester | ::{ flag=PH }:: [BronyPH](https://osu.ppy.sh/users/1492995) |
+| Map Pooler | ::{ flag=US }:: [Tekkito](https://osu.ppy.sh/users/7075211), ::{ flag=US }:: [D I O](https://osu.ppy.sh/users/3958619), ::{ flag=FR }:: [Nerowo](https://osu.ppy.sh/users/8642284), ::{ flag=US }:: [ChaosRaidz](https://osu.ppy.sh/users/3715823), ::{ flag=FR }:: [Ekoro](https://osu.ppy.sh/users/284905), ::{ flag=PH }:: [BronyPH](https://osu.ppy.sh/users/1492995) |
```
Comment (TPGPL) [inline wiki/Tournaments/The_Roundtable/en.md:35]:
```suggestion
| Mappooler | ::{ flag=US }:: [Tekkito](https://osu.ppy.sh/users/7075211), ::{ flag=US }:: [D I O](https://osu.ppy.sh/users/3958619), ::{ flag=FR }:: [Nerowo](https://osu.ppy.sh/users/8642284), ::{ flag=US }:: [ChaosRaidz](https://osu.ppy.sh/users/3715823), ::{ flag=FR }:: [Ekoro](https://osu.ppy.sh/users/284905), ::{ flag=PH }:: [BronyPH](https://osu.ppy.sh/users/1492995) |
```

File: wiki/Tournaments/The_Roundtable/en.md [hunk 4]
```diff
@@ -28,14 +28,19 @@ tags:
...
+| Map Pool Playtester | ::{ flag=US }:: [Tekkito](https://osu.ppy.sh/users/7075211), ::{ flag=PH }:: [BronyPH](https://osu.ppy.sh/users/1492995) |
```
Comment (TPGPL) [inline wiki/Tournaments/The_Roundtable/en.md:36]:
```suggestion
| Mappool playtester | ::{ flag=US }:: [Tekkito](https://osu.ppy.sh/users/7075211), ::{ flag=PH }:: [BronyPH](https://osu.ppy.sh/users/1492995) |
```

File: wiki/Tournaments/The_Roundtable/en.md [hunk 5]
```diff
@@ -28,14 +28,19 @@ tags:
 | :-- | :-- |
 | Host | ::{ flag=US }:: [BTMC](https://osu.ppy.sh/users/3171691) |
 | Project Manager | ::{ flag=US }:: [Kroytz](https://osu.ppy.sh/users/2339768), ::{ flag=US }:: [pishifat](https://osu.ppy.sh/users/3178418) |
```
Comment (TPGPL) [inline wiki/Tournaments/The_Roundtable/en.md:30]:
```suggestion
| Project manager | ::{ flag=US }:: [Kroytz](https://osu.ppy.sh/users/2339768), ::{ flag=US }:: [pishifat](https://osu.ppy.sh/users/3178418) |
```

**Review by TPGPL — DISMISSED** [review]

**Review by cl8n — APPROVED** [review]

**Review by peppy — APPROVED** [review]

**Pull request merged**